python/test/ngraph/util.py

```diff
@@ -45,25 +48,24 @@ def run_op_node(input_data, op_fun, *args):
     comp_args = []
     op_fun_args = []
     comp_inputs = []
-    for idx, data in enumerate(input_data):
-        if np.isscalar(data):
-            op_fun_args.append(ng.constant(data, _get_numpy_dtype(data)))
-        else:
-            node = ng.parameter(data.shape, name=ascii_uppercase[idx], dtype=data.dtype)
```

Comment:
Why are you removed this if-else block?
At the first glance it looks like you changed functionality and removed ng.parameter creation.
Please explain.

Reply:
After analysis of this utility function code and comparing it to what happens in Python API I came into conclusion that this `else` branch is simply unnecessary. Our Python API converts not only scalars but also `ndarray`'s into `Constant`s - take a look [here](https://github.com/NervanaSystems/ngraph/blob/05f2b9ef6b1a6a38bf892dade7b8f1b33444464e/python/ngraph/utils/types.py#L89-L104).

Reply:
Glad you asked for this. I should have explained it in PR description.